pandas/core/window.py

```diff
@@ -1044,18 +1055,26 @@ def validate(self):
                                           "for datetimelike and offset "
                                           "based windows")
 
+            if self.left_closed is not None and not is_bool(self.left_closed):
+                raise ValueError("left_closed must be a boolean")
+
             # this will raise ValueError on non-fixed freqs
             self.window = freq.nanos
+            if self.left_closed:
+                self.window += 1
```

Comment:
Can you add a test or two with values at finer resolutions than the offset to confirm this?  for example

```
In [11]: df = pd.DataFrame({'B': [1, 1, 1]},
    ...:                   index = [pd.Timestamp('20130101 09:00:30'),
    ...:                            pd.Timestamp('20130101 09:01:00'),
    ...:                            pd.Timestamp('20130101 09:02:00')])

In [12]: df.rolling('1Min').sum()
Out[12]: 
                       B
2013-01-01 09:00:30  1.0
2013-01-01 09:01:00  2.0
2013-01-01 09:02:00  1.0
```